hummingbot/cli/utils/symbol_fetcher.py

```diff
@@ -8,73 +8,90 @@
+                        return symbols
+                    except Exception:
+                        # Do nothing if the request fails -- there will be no autocomplete for ddex symbols
+                        return []
 
+    @staticmethod
+    async def fetch_radar_relay_symbols() -> List[str]:
+        symbols = set()
+        page_count = 1
+        while True:
+            async with aiohttp.ClientSession() as client:
+                async with client.get(f"{RADAR_RELAY_ENDPOINT}?perPage=100&page={page_count}", timeout=API_CALL_TIMEOUT) \
+                        as response:
+                    if response.status == 200:
+                        try:
+                            markets = await response.json()
+                            new_symbols = set(map(lambda symbol_details: symbol_details.get('id'), markets))
+                            if len(new_symbols) == 0:
+                                break
+                            else:
+                                symbols = symbols.union(new_symbols)
+                            page_count += 1
+                        except Exception:
+                            # Do nothing if the request fails -- there will be no autocomplete for radar symbols
+                            break
+        return list(symbols)
 
-async def fetch_radar_relay_symbols() -> List[str]:
-    async with aiohttp.ClientSession() as client:
-        async with client.get(RADAR_RELAY_ENDPOINT, timeout=API_CALL_TIMEOUT) as response:
-            if response.status == 200:
-                try:
-                    markets = await response.json()
-                    symbols = list(map(lambda symbol_details: symbol_details.get('id'), markets))
-                    return symbols
-                except Exception:
-                    # Do nothing if the request fails -- there will be no autocomplete for ddex symbols
-                    return []
+    @staticmethod
+    async def fetch_coinbase_pro_symbols() -> List[str]:
+        async with aiohttp.ClientSession() as client:
+            async with client.get(COINBASE_PRO_ENDPOINT, timeout=API_CALL_TIMEOUT) as response:
+                if response.status == 200:
+                    try:
+                        markets = await response.json()
+                        symbols = list(map(lambda symbol_details: symbol_details.get('id'), markets))
+                        return symbols
+                    except Exception:
+                        # Do nothing if the request fails -- there will be no autocomplete for coinbase symbols
+                        return []
 
+    async def fetch_all(self) -> Dict[str, List[str]]:
+        binance_symbols = await self.fetch_binance_symbols()
+        ddex_symbols = await self.fetch_ddex_symbols()
+        radar_relay_symbols = await self.fetch_radar_relay_symbols()
+        coinbase_pro_symbols = await self.fetch_coinbase_pro_symbols()
```

Comment:
use asyncio.gather?

Reply:
I will refactor this function later to fetch symbols for every exchange.